script/brave_license_helper.py

```diff
@@ -38,6 +38,34 @@ def AddBraveCredits(root, prune_paths, special_cases, prune_dirs,
         os.path.join('brave', 'third_party', 'libaddressinput'),
         os.path.join('brave', 'patches', 'third_party'),
 
+        # Duplicate notices already included in brave-core notices
+        os.path.join('brave', 'vendor', 'omaha', 'omaha', 'third_party',
+                     'chrome'),
+        os.path.join('brave', 'vendor', 'omaha', 'third_party', 'libzip'),
+        os.path.join('brave', 'vendor', 'omaha', 'third_party', 'lzma'),
+        os.path.join('brave', 'vendor', 'omaha', 'third_party', 'zlib'),
+
+        # Dependencies already included in the main breakpad notice.
+        os.path.join('brave', 'vendor', 'omaha', 'third_party', 'breakpad',
+                     'src', 'third_party', 'curl'),
+        os.path.join('brave', 'vendor', 'omaha', 'third_party', 'breakpad',
+                     'src', 'third_party', 'libdisasm'),
+        os.path.join('brave', 'vendor', 'omaha', 'third_party', 'breakpad',
+                     'src', 'third_party', 'linux'),
+        os.path.join('brave', 'vendor', 'omaha', 'third_party', 'breakpad',
+                     'src', 'third_party', 'mac_headers'),
+
+        # Dependencies which are no longer present in the codebase. This should
+        # be cleaned up in the upstream dependency including this.
+        os.path.join('brave', 'vendor', 'omaha', 'omaha', 'third_party',
+                     'hashlib'),
+
+        # No licensing information in recursive dependency. This should be fixed
+        # by the upstream dependency including this.
```

Comment:
```suggestion
        # No licensing information in recursive dependency.
```

Reply:
I do want to keep this one because in general "No licensing info" is no excuse for not including it.

Reply:
Ok, can we then rephrase it as `No licensing information in recursive dependency. This should be fixed by the upstream dependency`?

Reply:
That extra comment is there to try and avoid developers seeing this as an easy way to ignore build failures due to the licensing checks. :crossed_fingers:

Reply:
I'm just confused what "this" refers to here, in the second half of the sentence. Is it the same thing that's meant by the first "this" in the sentence?

Reply:
Good point. I've changed the wording to make it clearer. Removing the double use of "this".